### PR TITLE
Improve writing metadata for docx, pptx and odt

### PR DESCRIPTION
**Pull request by agusmba**
Currently the docx, pptx and odt writers have varying degrees of support for writing metadata:

* docx supports some core properties but lacks custom properties
* pptx supports a reduced set of core properties and also lacks custom properties
* odt supports some core properties and custom properties

This PR tries to address this shortcoming, by improving support for additional core properties and adding custom properties support where needed.

Previous attempts at independently fixing each writer can be seen [here](https://github.com/jgm/pandoc/pull/5238) and [here](https://github.com/jgm/pandoc/pull/5215), but this PR supersedes, unifies and builds on them.

The result is documented in the MANUAL (Metadata Variables chapter)

### CODE REVIEW

File: MANUAL.txt [hunk 1]
```diff
@@ -1381,14 +1381,48 @@ Metadata variables
...
+:   document subject, included in ODT, docx and pptx metadata
+
+`description`
+:   document description, included in ODT, docx and pptx metadata. Some
+    applications show this as `Comments` metadata.
+
+`category`
+:   document category, included in docx and pptx metadata
+
+`custom-properties`
```
Comment (jgm) [inline MANUAL.txt]: It's potentially a bit misleading to have `custom-properties` in this list as if it's also a variable or metadata field.
I'd suggest putting this statement about custom properties outside the list of variables.

**Comment by agusmba** [comment]
I guess the build error is due to not having .circleci config in my branch... Do you want me to rebase on current master?

**Comment by jgm** [comment]
I'll just merge it and it should be fine.

**Comment by jgm** [comment]
Thanks!

**Pull request merged**